cmd/relay-gossip/main.go

```diff
@@ -62,6 +62,16 @@ var peerWithFlag = &cli.StringSliceFlag{
 	Usage: "list of peers to connect with",
 }
 
+var httpFailoverFlag = &cli.StringSliceFlag{
```

Comment:
can this get added into the `cmd/client/lib` as part of common client construction?